frontend/model/contracts/group.js

```diff
@@ -507,7 +507,7 @@ DefineContract({
           }
 
           const groupIdToSwitch = Object.keys(contracts)
-            .find(cID => contracts[cID].type === 'group' &&
+            .find(cID => contracts[cID].type === 'gi.contracts/group' &&
```

Comment:
Hmm, this would be a complex refactor because the project has so many instances of "contracts", so if you'd like this change to be made, can you open an issue for it?

Reply:
If you think it's a good idea, then I'd be glad to (when I have time)